ci/azure-test-stable.yml

```diff
@@ -0,0 +1,33 @@
+jobs:
+- job: ${{ parameters.name }}
+  displayName: ${{ parameters.displayName }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  steps:
+  - template: azure-install-rust.yml
+    parameters:
+      platform: ${{parameters.name}}
+      rust_version: stable
+
+  - bash: |
+      export CI=True
+      cargo test --all
+    env:
+      LOOM_MAX_DURATION: 10
+    displayName: cargo test
+
+  - bash: |
+      sudo apt-get install gcc-multilib
+      rustup target add $TARGET
+      cargo check --all --exclude tokio-tls --target $TARGET
+      cargo check --tests --all --exclude tokio-tls --target $TARGET
+    condition: eq(variables['agent.os'], 'Linux')
+    env: { TARGET: 'i686-unknown-linux-gnu' }
+    displayName: cross compilation
+
+  - bash: |
```

Comment:
@carllerche The jobs are titled "$OS + Stable", but we're now doing explicit nightly testing. Do we want to have a different job? A different template?